decentraland/unity-explorer
# chore: added hover audio handler

## What does this PR change?

Here we add a very small component called HoverAudioHandler to be added to any UI element where we want to register hover events and play audio when the mouse enters them.
This is meant to simplify adding new sounds and clean up a bit some elements.

## How to test the changes?

Just add the component to any UI element, assign the audio config and play. When you hover over it whatever sound was configured should be heard.
